frontend/src/components/StatusTransfer.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect} from "react";
+import {
+    useAccount,
+    useContractRead
+} from "@starknet-react/core";
+import {TreasuryABI} from "../lib/treasuryABI";
+import CancelTransferBtn from "./CancelTransferBtn";
+import {formatBalance} from "../lib/erc20";
+import {TREASURY_ADDRESS} from "../lib/config";
+
+const StatusTransfer = () => {
+    const { address } = useAccount()
+    const {data, isLoading, refetch} = useContractRead({
+        functionName: 'get_live_transfers',
+        address: TREASURY_ADDRESS,
+        abi: TreasuryABI,
+        watch: false,
+        retry: false
+    })
+
+    const renderCancelBtn =  (status, transfer_id) =>
+        getTransferStatus(status) == 'PENDING' && address &&
+      <CancelTransferBtn transferId={transfer_id.toString()} />
+
+    const renderData = () => {
+        if (Array.isArray(data)) {
+            if (data.length === 0) {
+                return <div className='p-2'>There are no transfers</div>;
+            }
+
+            return data.map((transfer_item, index) => {
+
+                return (
+                    <div key={index} className="flex flex-wrap flex-row">
+                        <div className='flex flex-wrap flex-row w-full'>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider border-r border-slate-400">token</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">reciver</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 overflow-hidden">{transfer_item.token_addr.toString()}</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 overflow-hidden">{transfer_item.receiver.toString()}</div>
```

Comment:
Addresses are usually written in hex, and commonly are shortened (eg 0x3f37...fc60). They can be a link that leads to Starkscan or Voyager with the full address.